### PR TITLE
Add Knocki documentation

### PR DESCRIPTION
**Pull request by joostlek**
## Proposed change

Add Knocki documentation

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/119140
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/5578
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **Documentation**
  - Added instructions on setting up Knocki devices in Home Assistant to receive events triggered by these devices.

### CODE REVIEW

File: source/_integrations/knocki.markdown [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ha_codeowners:
+  - '@jgatto'
+  - '@joostlek'
+ha_domain: knocki
+ha_platforms:
+  - event
+ha_integration_type: integration
+---
+
+The Knocki integration can receive events for the triggers set up from your [Knocki devices](https://knocki.com/).
```
Comment (c0ffeeca7) [inline source/_integrations/knocki.markdown]:
```suggestion
The **Knocki** {% term integration %} can receive events for the triggers set up from your [Knocki devices](https://knocki.com/).
```

File: source/_integrations/knocki.markdown [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  - '@jgatto'
+  - '@joostlek'
+ha_domain: knocki
+ha_platforms:
+  - event
+ha_integration_type: integration
+---
+
+The **Knocki** {% term integration %} can receive events for the triggers set up from your [Knocki devices](https://knocki.com/).
+
```
Comment (c0ffeeca7) [inline source/_integrations/knocki.markdown:19]: Are there any prerequisites? For example: Do I need a Knocki account and login credentials when adding the integration?

**Comment by joostlek** [comment]
Usually I am a bit more verbal, but in this case I don't have a lot of words haha

File: source/_integrations/knocki.markdown [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+## Prerequisites
```
Comment (c0ffeeca7) [inline source/_integrations/knocki.markdown:20]:
```suggestion
## Prerequisites

```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @joostlek 👍

**Comment by c0ffeeca7** [comment]
@joostlek, can you take a look at the related brand PR?

**Comment by frenck** [comment]
Pinged Joost about the brands, merging this one in to I can start working on `rc`.

**Pull request merged**